woocommerce/woocommerce
# Update `create-extension` to work in monorepo context

### All Submissions:






### Changes proposed in this Pull Request:

In the context of the monorepo, `create-wc-extension` made little sense by placing an extension starter as a sibling to Woocommerce in the `plugins` folder. It's better as a separate project instead.

* Renamed to `create-extension`
* Places new example plugin as a sibling to the monorepo
* Uses `wp-env` to create a new dev environment loaded with Woocommerce
* Updates dependencies

### How to test the changes in this Pull Request:

1. `pnpm install`
2. `pnpm run create-extension` and follow the prompts
3. `cd ../<project-name>`
4. `wp-env start`
5. `pnpm install`
6. `pnpm start`

Check out your new `wp-env` install and see a React powered WooCommerce Admin page titled "My Example Page".

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### FOR PR REVIEWER ONLY:
